src/main/scala/org/constellation/infrastructure/endpoints/SnapshotEndpoints.scala

```diff
@@ -193,15 +200,17 @@ class SnapshotEndpoints[F[_]](implicit F: Concurrent[F]) extends Http4sDsl[F] {
             case Invalid(IncorrectSenderId(_))                                                  => Response[F](status = Unauthorized).pure[F]
             case Invalid(e)                                                                     => logger.error(e)(e.getMessage) >> InternalServerError()
             case Valid(_) =>
+              val processProposalAsync = F.start(for {
+                _ <- C.shift
```

Comment:
please ensure that you are shifting on the same thread pool. I mean, the thread pool where this process is being executed and thread pool assigned to `C: ContextShift` should be the same. otherwise you will switch thread pools unexpectedly

Reply:
Ok, I will check that in logs and come back with the answer 👍

Reply:
Ok, everything happens still on the `unbounded-pool-1`